var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -50,7 +50,10 @@ class Gptune(CMakePackage):
 
     depends_on('superlu-dist@develop', when='+app', type=('build', 'run'))
 
-    conflicts('openmpi@:3')
+    depends_on('openmpi@4:', when='+openmpi', type=('build', 'run'))
+    conflicts('mpich', when='+openmpi')
+    conflicts('spectrum-mpi', when='+openmpi')
+    conflicts('cray-mpich', when='+openmpi')
```

Comment:
Why a `(build, run)` dependency for `openmpi`? Assuming `openmpi` is the only viable choice, I think this can be written in a more robust way as:
```suggestion
    depends_on('openmpi@4:', when='+openmpi')
    depends_on('mpi', when='+openmpi')
```
since the `mpi` dependency will rule out any other `mpi` provider.

Reply:
@alalazo mpi is a mandatory requirement of gptune. 

If the variant +openmpi is enabled, gptune needs openmpi, otherwise, gptune can use mpich, spectrum-mpi, cray-mpich, openmpi or other mpi providers. 

So what's the appropriate way of specifying this?

Reply:
But the variant should be named "openmpionly" to avoid confusion.

Reply:
I've changed the variant name to "mpispawn"